### DISCUSSION TITLE
Mailu admin stuck at alembic.runtime.migration

### DISCUSSION THREAD

**Discussion by pr0ton11** [Mailu/Mailu#3020] [User support] [answered]
## Environment & Version

* Podman version: 4.3.1
* Podman-compose version 1.0.6
* Debian 12
* Latest version of Mailu (the bug is caused by podman-compose --pull)

## Description

In an effort to update my system I ran podman-compose --pull and recreated the containers. Now the admin container is stuck at alembic.runtime.migration steps and does not come up.
Upgraded from Mailu 2.0.x to 2.0.31

## Replication Steps / Observed behaviour

* Run podman-compose up -d --force-recreate --pull
* The mailu-admin container does not come back up as healthy

## Expected behaviour

* Run podman-compose up -d --force-recreate --pull
* The mailu-admin container comes back as healthy and all works

## Logs

```
root@redacted:/var/mailu/data# podman logs mailu_admin_1
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
```

**Comment by ghostwheel42** [comment, marked as answer]
This reads like https://github.com/Mailu/Mailu/issues/2992#issuecomment-1773908866

**Reply by pr0ton11** [reply]
#1447 describes why this happens. I know that podman is an unsupported configuration, but it's not always possible to run docker in every setup.
I would really like to see the devs not intentionally breaking working setups with rot13 encrypted checks for docker and / or wierd environment variables. Especially since the condescending message (linking how to ask questions the smart way, really?) is not even shown in the log messages (did someone even test if this works?)
You only make people create more pointless issues in your repo by doing it in this manner.
